addOns/ascanrules/src/test/java/org/zaproxy/zap/extension/ascanrules/TestCrossSiteScriptV2UnitTest.java

```diff
@@ -718,14 +925,15 @@ protected Response serve(IHTTPSession session) {
 
         this.nano.addHandler(handler);
 
+        // When
         HttpMessage msg = this.getHttpMessage(test + "?name=test");
         msg.getRequestHeader().setMethod(HttpRequestHeader.PUT);
 
         rule.setConfig(new ZapXmlConfiguration());
         this.rule.setAlertThreshold(AlertThreshold.MEDIUM);
         this.rule.init(msg, this.parent);
-        // When
```

Comment:
Why was this changed, 721-726 is all setup (given)

Reply:
I have reset the changed lines.